src/lib/components/FormModel/children/Form/UI/Select/select.test.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { render, cleanup } from 'react-testing-library';
+import { render, cleanup } from '@testing-library/react';
```

Comment:
In this unit test, as in some others featured in this PR (namely all the tests where the `render` call hasn't been moved), `render` is still called outside of the scope of the test: 
```js
describe('Provider Login Container', () => {
  const { container } = render(<Select />);

  it('shoud renders without crashing', () => {
    expect(container).toBeTruthy();
  });
});
```
instead of 
```js
describe('Provider Login Container', () => {
  it('shoud renders without crashing', () => {
    const { container } = render(<Select />);
    expect(container).toBeTruthy();
  });
});
```
This doesn't create any test failures, but if these tests were to be extended in the future, they would have to be refactored to be aligned with the others, or the same issues may arise. Is this something that should be addressed now ?

Reply:
That's a good question - I'd argue no, that's beyond the scope of this ticket. I think all our tests across the board could use a refresh. Some of these are very old tests (relative to the project).